packages/manager/src/managers/versions/VersionsManager.ts

```diff
@@ -53,28 +88,7 @@ export class VersionsManager extends BaseManager {
 
 	async getRunningSliceMachineVersion(): Promise<string> {
 		const sliceMachineDir = await this.project.locateSliceMachineUIDir();
-
-		const sliceMachinePackageJSONContents = await fs.readFile(
-			path.join(sliceMachineDir, "package.json"),
-			"utf8",
-		);
-
-		let sliceMachinePackageJSON: unknown;
-		try {
-			sliceMachinePackageJSON = JSON.parse(sliceMachinePackageJSONContents);
-		} catch {
-			// noop
-		}
-
-		const { value, error } = decodePackageJSON(sliceMachinePackageJSON);
-
-		if (error) {
-			throw new Error(
-				`Invalid ${SLICE_MACHINE_NPM_PACKAGE_NAME} \`package.json\` file. ${error.errors.join(
-					", ",
-				)}`,
-			);
-		}
+		const value = await readPackageJSON(sliceMachineDir);
```

Comment:
Small suggestion: rename `value` (a generic name) to `packageJSON` (a descriptive name).

Reply:
Done, thanks 🙂